rest/model/patch.go

```diff
@@ -151,12 +152,19 @@ func (apiPatch *APIPatch) GetCommitQueuePosition() error {
 }
 
 func (apiPatch *APIPatch) GetIdentifier() {
-	if apiPatch.ProjectIdentifier != nil {
+	if utility.FromStringPtr(apiPatch.ProjectIdentifier) != "" {
 		return
 	}
 	if utility.FromStringPtr(apiPatch.ProjectId) != "" {
 		identifier, err := model.GetIdentifierForProject(utility.FromStringPtr(apiPatch.ProjectId))
-		if err == nil {
+		if err != nil {
+			grip.Error(message.WrapError(err, message.Fields{
```

Comment:
Is this meant to be temporary debug logging for fixing this one issue, or permanent? If its temporary, can we add a ticket to remove it? Otherwise, if it's permanent, will this be useful for tracing down issues in the future? It seems like this is deep enough in the implementation logic that nobody would think to look for this log.

Reply:
I meant to keep it there, with my logic being that it's good for us to have access to it, because right now there is not much traceability.

Reply:
That sounds fine to me. In that case, we can remove the `if err != nil {` check because `grip.Error(message.WrapError(err...` already has internal logic to not log if the err is nil.

Reply:
thanks, removed